.github/workflows/cd.yaml

```
@@ -53,3 +53,20 @@ jobs:
           platforms: linux/amd64
           push: true
           tags: ${{ steps.image-tags.outputs.result }}
+  docs:
+    name: Deploy docs
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        # "ref" specifies the branch to check out.
+        # "github.event.release.target_commitish" is a global variable and specifies the branch the release targeted
+        ref: ${{ github.event.release.target_commitish }}
+
+    # Deploy docs
+    - name: Deploy docs
+      uses: mhausenblas/mkdocs-deploy-gh-pages@master
```

Should we pin this? Looks like mkdocs maintainers aren't involved with it, so I usually err on the side of caution in these cases
```suggestion
      uses: mhausenblas/mkdocs-deploy-gh-pages@1.26
```

---

Yeah, it would nice to not have stuff break if there are significant changes.